src/components/Layout/Header/NavBar/Native/BackupPassphraseModal/BackupPassphraseInfo.tsx

```diff
@@ -0,0 +1,142 @@
+import { ArrowBackIcon } from '@chakra-ui/icons'
+import { ModalCloseButton } from '@chakra-ui/modal'
+import {
+  Alert,
+  AlertDescription,
+  AlertIcon,
+  Button,
+  Code,
+  IconButton,
+  ModalBody,
+  ModalFooter,
+  ModalHeader,
+  Tag,
+  useColorModeValue,
+  Wrap,
+} from '@chakra-ui/react'
+import { Vault } from '@shapeshiftoss/hdwallet-native-vault'
+import range from 'lodash/range'
+import { ReactNode, useEffect, useMemo, useState } from 'react'
+import { FaEye } from 'react-icons/fa'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router'
+import { SlideTransition } from 'components/SlideTransition'
+import { Text } from 'components/Text'
+import { logger } from 'lib/logger'
+
+import { BackupPassphraseRoutes } from './BackupPassphraseCommon'
+
+const moduleLogger = logger.child({
+  namespace: ['Layout', 'Header', 'NavBar', 'Native', 'BackupNativePassphrase'],
+})
+
+export const BackupPassphraseInfo = ({ vault }: { vault: Vault | null }) => {
+  const translate = useTranslate()
+  const history = useHistory()
+  const [revealed, setRevealed] = useState<boolean>(false)
+  const handleShow = () => {
+    setRevealed(!revealed)
+  }
+  const [words, setWords] = useState<ReactNode[] | null>(null)
+  const alertColor = useColorModeValue('blue.500', 'blue.200')
+  useEffect(() => {
+    if (!vault) return
+    ;(async () => {
+      try {
+        setWords(
+          (await vault.unwrap().get('#mnemonic')).split(' ').map((word: string, index: number) => (
+            <Tag
+              p={2}
+              flexBasis='31%'
+              justifyContent='flex-start'
+              fontSize='md'
+              key={word}
```

Comment:
Usually an antipattern, but I'd rather use the index as a key here for security reasons. At this point the words are already in clear in the browser, and rendered as a children but still, the less places they are in the better